css/style.css

```diff
@@ -0,0 +1,892 @@
+body {
+  width: 100%;
+  height: 100%;
+  font-family: Montserrat;
+  font-style: normal;
+  color: #000;
+  margin: 0;
+}
+
+/* header*/
+
+.header {
+  max-width: 100%;
+  height: 50px;
+  margin: 0 auto;
+  margin: 24px 60px 60px 42px;
+}
+
+.inner {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.logo {
+  width: 166px;
+  height: 51px;
+}
+
+.tesla-logo {
+  width: 100%;
+}
+
+.products {
+  width: 490px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.menu {
+  width: 200px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.header__link {
+  font-weight: 600;
+  font-size: 15px;
+  text-decoration: none;
+  color: #000;
+  transition: color 0.3s ease-out;
+}
+
+.header__link:hover {
+  color: #E51937;
```

Comment:
Codacy found an issue: [Expected "#E51937" to be "#e51937" (color-hex-case)](https://app.codacy.com/gh/WebHeroSchool/finalproject/pullRequest?prid=5923515)